specs/kyber/src/sampling.rs

```diff
@@ -149,8 +150,7 @@ fn sum_coins(coins: &mut BitVectorChunks<'_>) -> KyberFieldElement {
 pub fn sample_poly_cbd(eta: usize, bytes: &[u8]) -> KyberPolynomialRingElement {
     assert_eq!(bytes.len(), eta * 64);
 
-    // b ← BytesToBits(B)
-    let bits: BitVector = bytes.into();
+    let bits = bytes_to_bits(bytes);
 
     let mut bits = bits.chunks(eta);
 
```

Comment:
Ugh, where is the `sum_coins` coming from? Did I miss this in the last review.
That probably won't work. hax might be able to deal with it by now, but we'll have to look at it tomorrow when extracting.

Please stop force pushing. That breaks the review workflow and have been a reason for this.

Reply:
I'll refrain from force pushes, my bad!